frontend/src/components/common/RoleRoute.tsx

```diff
@@ -37,7 +37,11 @@ const RoleRoute = ({
   );
 
   if (!userHasRequiredRole) {
-    return <AccessDenied />;
+    return (
```

Comment:
Do we have **Analytics** Access Denied Page?

Reply:
added.